.prettierignore

```diff
@@ -1,31 +1,39 @@
-node_modules
-**/.next/**
-**/_next/**
-**/dist/**
+# Build artifacts
+.next/
+.turbo/
+_next/
+__tmp__/
+dist/
+node_modules/
+target/
+compiled/
+
+lerna.json
+test-timings.json
+pnpm-lock.yaml
+
 packages/next/src/bundles/webpack/packages/*.runtime.js
 packages/next/src/bundles/webpack/packages/lazy-compilation-*.js
-packages/next/src/compiled/**
-packages/react-refresh-utils/**/*.js
-packages/react-refresh-utils/**/*.d.ts
-packages/react-dev-overlay/lib/**
-**/__tmp__/**
-lerna.json
+
 .github/actions/next-stats-action/.work
 .github/actions/issue-validator/index.mjs
 .github/actions/issue-labeler/lib/index.js
-packages/next-swc/crates/next-core/js/src/compiled/**/*
+
 packages/next-swc/crates/**/tests/**/output*
-packages/next-swc/target/**/*
+packages/next-swc/crates/core/tests/loader/issue-32553/input.js
```

Comment:
I put `prettier-ignore` comments on every single line of this file and prettier still broke the lines, so I had to add it here

Reply:
In the files where it works the exclusion only applies to a single line, which I think is preferable